torchci/rockset/prodVersions.json

```diff
@@ -62,4 +62,4 @@
     "workflow_duration_percentile": "0b28cf65de7cfe07",
     "workflow_load": "eff394d8e0b76436"
   }
-}
\ No newline at end of file
+}
```

Comment:
Did you hand edit this file?  Surprised to see a new-line getting added by our tools